### PR TITLE
update tranco list and require tests during pr

### PR DESCRIPTION
**Pull request by mindofmar**
This PR accomplishes some changes that @409H, @0xOhm and I discussed last week as part of a greater effort to reduce false positives.

Tasks completed:
- [x] Updating tranco list to latest list and expanding it to top 200,000 sites rather than 100,000
- [x] CICD now requires tests to run. If a new addition to the blocklist is detected within the `trancos.txt` file, CICD will fail and require manual review.
     - If it is a false positive, it must be removed from the submission list
     - If it is true positive but the URL is on the `tracos.txt` list, it must be added to the override list within the `confirm-blocklist.test.js` file in order to make the test pass. It is the responsibility of the submitter (and ideally code owners need to review at this point) to verify that this URL should be on the blocklist. 

I would like feedback on:
- What is the preferred method for submitters to receive notifications when one of their PRs fails CICD? (how does this work if it is a bot like ChainPatrol or SEAL?)

### CODE REVIEW

File: src/config.json
Comment (mindofmar) [inline src/config.json:51554]: The DNS records of `pancakeswap.com` and `pancakeswap.finance` appear to be the same, which is the reason for the removal here. (the .com also redirects to the correct site)

**Review by NEllusion — COMMENTED** [review]
On the topic of CI, the reason why CI fails on all commits in main is due to the fact that the following github action fails: https://github.com/MetaMask/eth-phishing-detect/blob/main/.github/workflows/auto-cleanup.yml

Perhaps we can delete this at the same time to that we can enforce CI

**Comment by mindofmar** [comment]
> On the topic of CI, the reason why CI fails on all commits in main is due to the fact that the following github action fails: https://github.com/MetaMask/eth-phishing-detect/blob/main/.github/workflows/auto-cleanup.yml
> 
> Perhaps we can delete this at the same time to that we can enforce CI

Done, this step is now removed since it always fails

File: test/update-static-lists.js
Comment (NEllusion) [inline test/update-static-lists.js:9]: I could be misinterpreting this, but I see we are using the 200k list, but tracos.txt is ~300k lines long. Is this expected?

File: .github/CODEOWNERS
Comment (NEllusion) [inline .github/CODEOWNERS]: It may make sense to add product safety team as codeowners in other areas such as `test/` as well

**Review by NEllusion — COMMENTED** [review]
> What is the preferred method for submitters to receive notifications when one of their PRs fails CICD? (how does this work if it is a bot like ChainPatrol or SEAL?)

For user accounts, this is up to them to configure their notifications in their settings: 

However, github also exposes the ability for them to programatically detect when one of their PRs is failing a status check, it may be preferable for these devs to check for failures that way and fire off their own notifications. 

As this will introduce a potentially breaking change, it may make sense to warn our main automated contributors prior to enabling the enforcement

File: .github/CODEOWNERS
Comment (mindofmar) [reply .github/CODEOWNERS]: Done in next commit

File: test/update-static-lists.js
Comment (mindofmar) [reply test/update-static-lists.js:9]: The list is 200k lines, but the diff in the PR makes it look like it is 300k

Comment (NEllusion) [reply test/update-static-lists.js:9]: @mindofmar ahh makes sense, my bad! LGTM then once merge conflicts are resolved

**Review by AugmentedMode — APPROVED** [review]

**Review by 409H — APPROVED** [review]

**Pull request merged**
